### ISSUE TITLE
Is this really working ?

### ISSUE THREAD

**Issue by JobiJoba** [rationalappdev/react-native-drag-and-drop-tags-tutorial#5]
Hi, 

I'm using expo SDK 28 (so it's react 0.55 if I'm not wrong) and I try to incorporate that in my application. 
In fact, I don't need a lot of those feature as I'm using it to make a small "game". 

A sentence is cut in multiple words, and each word is display as a "tag" in a wrong order, and then people have to put it in the correct order to go on the next sentence. 

So all of that, to say that I remove the functionality to add / remove a tag, no modal etc ... 

The issue is that when I press a tag, he's selecting another one (on real android phone) or simulator. 
On iOS it's even worse, it does not do anything at all (I tried with the remove first and I had to remove multiple tag before it's start doing the drag & drop stuff) --> looking in the method onMoveShouldSetPanResponder ; the line  const tag = this.findTagAtCoordinates(moveX, moveY); never find a tag.

I'll try to do some more research on that, but if someone already find the solution ... ;) would be nice to share. 

Thx

**Comment by yuriykostin14** [comment]
Hello.

Anyone found solution regarding this issue?!

**Comment by kockok** [comment]
Yes, I am encountering the same situation, the pressed tag is not highlighted whilst the nearby one does...
